contributors/chairs-and-techleads/leadership-changes.md

```diff
@@ -0,0 +1,27 @@
+# Leadership Changes
+
+This document covers steps needed to propose changes to SIG/WG/UG leadership.
+
+- [ ] Discuss the proposed changes with the current leadership.
+
+- [ ] Send an email to the SIG/WG/UG mailing list and cc the [kubernetes-dev]
+mailing list. At a minimum, the email should contain:
+  - [ ] intent to step down as the current lead
+  - [ ] if nominating another lead
+    - [ ] 1-2 lines about why they are being nominated
+    - [ ] contacts (usually current leads) to privately reach out
+          to for questions or concerns about the nomination
+  - [ ] if this was discussed in a meeting, link to meeting notes
+  - [ ] lazy consensus deadline of at least 2 business days
+
+- [ ] Once lazy consensus has been achieved, submit a PR to update
+[`sigs.yaml`] and use the [generator doc] to update `README.md` and
+`OWNERS_ALIASES` files.
+
+- [ ] Create an issue in the kubernetes/community repo to track
+completion of the [Inclusive Leadership Training] by the new lead(s).
```

Comment:
seems like the mechanics should come _before_ updating sigs.yaml:

* org member
* completed required training

Reply:
> completed required training

We ask leads to complete the training within 30 days of their appointment but I added it as a prerequisite here to try enforcing it as a best practice 

https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#all-leads